src/aliceVision/image/io.cpp

```diff
@@ -509,15 +509,13 @@ void writeImage(const std::string& path,
   }
   else if((imageColorSpace != EImageColorSpace::LINEAR) && (imageColorSpace != EImageColorSpace::NO_CONVERSION)) // ACES or ACEScg
   {
-      char const* val = getenv("ALICEVISION_ROOT");
-      if (val == NULL)
+      const auto& root = getAliceVisionRoot();
```

Comment:
I'm always overthinking simple things ;-), but maybe having a function also for config.ocio, like
```c++
std::string getAliceVisionOCIOConfig()
{
    if (!getAliceVisionRoot().empty())
        return getAliceVisionRoot() + "/share/aliceVision/config.ocio";
    return  {};
}
```
It's just that if for whatever reason it needs to be used in another place we have the path only in one place. What do you think?

Reply:
I think this makes sense. Fixed now.